app/konnect/dev-portal/access-and-approval/add-teams.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Add Developer Teams from Identity Providers
+content_type: how-to
+---
+
+In the {{site.konnect_short_name}} Dev Portal, you can import existing developer teams from a third-party identity provider (IdP) and map their permissions to elements in {{site.konnect_short_name}}. This allows you to import developers automatically instead of manually copying over every single team of developers from your IdP to {{site.konnect_short_name}}.
+
+This guide explains how to map the permissions, including scopes and claims, from your group of developers in your IdP to your organization's team in {{site.konnect_short_name}}. Although this guide uses Okta, Azure, and Auth0 as examples, you can use any IdP that conforms to OIDC standards. 
+
+## Prerequisites
+An IdP configured in Konnect (OIDC or DCR) for [application registration](konnect/dev-portal/applications/enable-app-reg/)
+
+## Create developer teams in your IdP
+
+<!-- do these IdPs need the Kong callback URl added to them to be properly configured? Or is this just a one-way pull from the IdP to Konnect, so it doesn't need any information from Konnect?-->
+
+{% navtabs %}
+{% navtab Okta %}
+In Okta, create a [new group of developers](https://help.okta.com/asa/en-us/content/topics/adv_server_access/docs/setup/create-a-group.htm) that you want to map to {{site.konnect_short_name}}. Be sure to configure the team settings and attributes as needed.
+{% endnavtab %}
+{% navtab Azure %}
+In Azure, create a [new team of developers](https://learn.microsoft.com/azure/devops/organizations/settings/add-teams?view=azure-devops&tabs=preview-page) that you want to map to {{site.konnect_short_name}}.
+{% endnavtab %}
+{% navtab Auth0 %}
+In Auth0, create a [new team of developers](https://auth0.com/docs/get-started/tenant-settings/auth0-teams) that you want to map to {{site.konnect_short_name}}.
+{% endnavtab %}
+{% endnavtabs %}
+
+## Map IdP developer teams in {{site.konnect_short_name}}
+
+1. In {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal), click **Settings**.
+
+1. In the **General** setting tab, enable **Portal RBAC**.
+    
+    Enabling RBAC allows you to create teams in {{site.konnect_short_name}}. You can disable RBAC after you map teams from your IdP if you don't want to use it.
+
+1. From the **Teams** settings in the side bar, click **New Team** and configure the team.
+
+1. From the IdP team you just created, click the **Products** tab and click **Add Roles**. This allows you to assign API products and the role for the API product to members of your IdP team.
+
+1. From **Settings** in the Dev Portal side bar, click the **Team Mappings** tab and then select **IdP Mapping Enabled**. 
+
+1. Click **Configure OIDC provider** and configure the settings using the following mappings:
```

Comment:
can we copy the same language from here: https://docs.konghq.com/konnect/dev-portal/applications/dynamic-client-registration/auth0/#configure-the-dev-portal? The user may have multiple applications in their IdP so it does not make sense imo to say "copy over the application ID/issuer for your IdP", it's more like copy over the application ID/issuer from the Application you've configured in your IdP to manage your teams.

Then the Client ID instructions is more like: "Enter the Client ID from the admin application created in Auth0 above into the Initial Client ID field."

same for Provider URL and Client Secret